tests/exporters/exporters_utils.py

```diff
@@ -59,6 +59,7 @@
     "gptj": "hf-internal-testing/tiny-random-GPTJModel",
     "groupvit": "hf-internal-testing/tiny-random-groupvit",
     "ibert": "hf-internal-testing/tiny-random-IBertModel",
+    "imagegpt": "hf-internal-testing/tiny-random-ImageGPTModel",
```

Comment:
Could you also add an entry below in `PYTORCH_EXPORT_MODELS_LARGE` for [this model](https://huggingface.co/openai/imagegpt-small) for instance?

Reply:
fixed